katas/es6/language/generator/yield.js

```diff
@@ -0,0 +1,43 @@
+// Generators - Yield Expressions
+// To do: make all tests pass, leave the assert lines unchanged!
+
+describe('generator instance', () => {
+  
+  function* generator() {
+    yield 'hello';
+    yield 'world';
+  }
+  
+  let genInstance;
+
+  beforeEach(function() {
+    genInstance = generator();
+  });
+  
+  it('instance converted to an array returns all yielded values', () => {
```

Comment:
and 
`it('converting the generator to an array resumes the generator until all values got received')`
or something like that ... so that the person reading the test description knows what to expect and in the best case, knows how to implement the test code ... 
Warning: I might have a very certain style already creating katas ... sorry if I am too pushy in one direction